package.json

```diff
@@ -369,6 +372,7 @@
       "safe-json-stringify"
     ],
     "onlyBuiltDependencies": [
+      "@ast-grep/lang-ruby",
```

Comment:
i think this needs to be added to this file too
https://github.com/renovatebot/renovate/blob/main/test/e2e/package.json#L9